src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -140,6 +152,20 @@ public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
             .collect(Collectors.toCollection(ArrayList::new));
     }
 
+    private static String getVenueID(SscsCaseData caseData, String nextHearingVenue) {
+        if (SOMEWHERE_ELSE.equals(nextHearingVenue)) {
+            return caseData.getAdjournCaseNextHearingVenueSelected().getValue().getCode();
+        } else if (SAME_VENUE.equals(nextHearingVenue)) {
+            Hearing latestHearing = caseData.getLatestHearing();
+
+            if (nonNull(latestHearing)) {
+                return latestHearing.getValue().getVenueId();
+            }
```

Comment:
I think there should be an else clause here for if latestHearing is null, throw a different exception to the one at the end of the method

Reply:
I think the one exception is descriptive enough to cover both cases and keeping just the one makes it look cleaner

Reply:
Done